### PR TITLE
Dynamite updates

### PR DESCRIPTION
**Pull request by adrianlee44**
- Update node packages to latest versions
- Update UpdateBuilder and PutItemBuilder to allow ReturnValues NONE
- Fix type checking

cc @xiao for PR review.

### CODE REVIEW

File: lib/PutItemBuilder.js [hunk 1]
```diff
@@ -12,9 +12,17 @@ var typeUtil = require('./typeUtil')
...
 function PutItemBuilder(options) {
   Builder.call(this, options)
+
+  /** @private {string} */
+  this._returnValues = 'ALL_OLD'
 }
 require('util').inherits(PutItemBuilder, Builder)
 
+PutItemBuilder.prototype.setReturnValues = function (val) {
+  this._returnValues = val
```
Comment (xiao) [inline lib/PutItemBuilder.js]: Should we throw an error for values that we don't support?

**Comment by xiao** [comment]
LGTM!

**Comment by adrianlee44** [comment]
Will resolve CI issue before merging.

File: lib/PutItemBuilder.js (hunk 1 above)
Comment (adrianlee44) [inline lib/PutItemBuilder.js]: I think that's a good idea. Do you think we should have an enum mapping somewhere?

Comment (adrianlee44) [inline lib/PutItemBuilder.js]: Will add support for this in a follow up PR.

**Pull request merged**